pages/index.js

```diff
@@ -1,7 +1,10 @@
+import RecipeList from "@/components/RecipeList";
+
 export default function HomePage() {
   return (
     <div>
-      <h1>Hello from Next.js</h1>
+      <h1>Fork Yeah!</h1>
+      <RecipeList />
     </div>
```

Comment:
Consider adding descriptive semantic elements, for example a `<main>` which encapsulate the primary content instead of the generic `<div>` 🙃